src/app/current-users/current-users.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { Users } from './users';
+import { UsersObject } from '../../../api-objects/UsersObject';
+
+@Component({
+  selector: 'current-users',
+  templateUrl: './current-users.component.html',
+  styleUrls: ['./current-users.component.css']
+})
+
+export class CurrentUsersComponent implements OnInit {
+  public usersTable: UsersObject[] = [];
+  public pageCounter: number = 0;
+
+  constructor(/*private currentusersTableService: GetCurrentUsersService*/) { }
```

Comment:
When we add the service. It will be Liam's `Account Admin Service`

Reply:
I thought Liam was doing patients, and Dominic was the one doing users.

Reply:
Yup, my bad. The Admin Service should be ready to test then!